### PR TITLE
update parameters name send from Danse

### PR DESCRIPTION
**Pull request by 4D42**
Update parameters names send from Danse

### CODE REVIEW

File: Suite2p/suite2p_parameters.py [hunk 1]
```diff
@@ -23,12 +23,18 @@ def __init__(self, danse_params: dict):
...
-        self.ops['threshold_scaling'] = self.params['Threshold'] # Threshold for ROIs detection
-        self.ops['tau']               = self.params['DecayTime'] # Timescale of gcamp to use for deconvolution
+        self.ops['threshold_scaling'] = self.params['ROIsThreshold'] # Threshold for ROIs detection
+        self.ops['tau']               = self.params['BiosensorDecayTime'] # Timescale of gcamp to use for deconvolution
+        self.ops['diameter']          = self.params['Diameter'] # (cellpose)
+
         self.ops['nplanes']           = len(self.paths[defs.Parameters.Path.H5PATH])
         self.ops['data_path']         = [self.paths[defs.Parameters.Path.TMP_DIR]]
 
+        self.ops['batch_size']        = 50 # Decrease the batch_size in case low RAM on computer
```
Comment (doriclenses) [inline Suite2p/suite2p_parameters.py:33]: I'm not sure about that.. If the computer is good, the processing would be just long for nothing

Comment (doriclenses) [reply Suite2p/suite2p_parameters.py:32]: Yepp, for now I'd keep the default 500. We can try increase/decrease this value depending on the RAM later

Comment (4D42) [reply Suite2p/suite2p_parameters.py:32]: 500 is too much for voluscan :/ 

100 is good on my machine. I can check for 200 how it go

File: Suite2p/suite2p_parameters.py [hunk 2]
```diff
@@ -20,14 +20,30 @@ def __init__(self, danse_params: dict):
...
+        # Suite2p Main Settings
         self.ops['data_path']         = [self.paths[defs.Parameters.Path.TMP_DIR]]
+        self.ops['nplanes']           = len(self.paths[defs.Parameters.Path.H5PATH])
+        self.ops['tau']               = self.params['BiosensorDecayTime'] # Timescale of GCaMP to use for deconvolution
+
+        # Suite2p Registration Settings
+        self.ops['batch_size']        = 500 # Decrease the batch_size in case low RAM on computer
+        self.ops['smooth_sigma']      = 4   # STD in pixels of the gaussian used to smooth the phase correlation between the reference image and the frame which is being registered
+    
+        # Suite2p 1P registration
```
Comment (doriclenses) [inline Suite2p/suite2p_parameters.py:35]: Suite2p is designed to 2-photon microscope recordings, both voluscan and our microscope are 1-photon, so we kinda have to enable these parameters. Not sure about the values though, we should probably do an estimation based on the neuron diameter and image size. @4D42 @kapilseh what do you think?

File: Suite2p/suite2p_parameters.py (hunk 1 above)
Comment (doriclenses) [reply Suite2p/suite2p_parameters.py]: oki let's update to 100

**Review by doriclenses — APPROVED** [review]

File: Suite2p/suite2p_parameters.py (hunk 2 above)
Comment (4D42) [reply Suite2p/suite2p_parameters.py:35]: Yes true.

**Review by kapilseh — APPROVED** [review]

**Pull request merged**